src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -385,6 +385,40 @@ public void StartupHookSupport_is_false_by_default_on_trimmed_apps(string target
             }
         }
 
+        [Theory]
+        [InlineData("net5.0")]
+        [InlineData("net6.0")]
+        public void CustomResourceTypesSupport_is_false_by_default_on_trimmed_apps(string targetFramework)
```

Comment:
Since this is basically duplicated with the test above it, can we just add the new checks in the existing test?